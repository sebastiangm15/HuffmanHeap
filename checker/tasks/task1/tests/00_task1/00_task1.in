5
4 B4
7 A7
7 C7
12 D12
16 E16
