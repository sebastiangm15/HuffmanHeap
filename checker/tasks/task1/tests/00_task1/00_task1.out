46-C7B4A7D12E16 
18-C7B4A7 28-D12E16 
7-C7 11-B4A7 12-D12 16-E16 
4-B4 7-A7 
